packages/package_bumbershoot_3_0_8738/tool_dependencies.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="bumbershoot" version="3.0.8738">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://teamcity.labkey.org/guestAuth/repository/download/ProteoWizard_Bumbershoot_Linux_X86_64/265091:id/bumbershoot-bin-linux-gcc48-release-3_0_8738.tar.bz2</action>
```

Comment:
This will not help. The Galaxy project is working on a more scalable solution. This was my poor many approach and it is serving me well. But I see the end ... maybe a second repo ;)